### DISCUSSION TITLE
请问这个UI使用的是什么框架？

### DISCUSSION THREAD

**Discussion by ccmywish** [Planshit/Tai#146] [开发者闲聊]
对C#的生态不太了解，只知道现在有很多种UI框架可以使用。

这个App做的非常的好看，好奇使用的是哪一个呢？如果要开启新项目，建议使用C#的哪一个框架？

**Comment by noberumotto** [comment]
没有使用UI框架，是手写的样式。UI框架的话可以推荐一下这个：https://github.com/lepoco/wpfui
